packages/remix-dev/config.ts

```diff
@@ -727,3 +732,4 @@ let listFormat = new Intl.ListFormat("en", {
 });
 
 export let serverBuildTargetWarning = `The "serverBuildTarget" config option is deprecated. Use a combination of "publicPath", "serverBuildPath", "serverConditions", "serverDependenciesToBundle", "serverMainFields", "serverMinify", "serverModuleFormat" and/or "serverPlatform" instead.`;
+export let flatRoutesWarning = `The old route convention has been deprecated in favor of "flat routes". Please enable it via your remix.config https://remix.run/docs/en/main/file-conventions/route-files-v2`;
```

Comment:
Should we prefix this with `⚠️ DEPRECATED: or something to make it stand out a bit more?

```suggestion
export let flatRoutesWarning = `⚠️ DEPRECATED: The old nested folders route convention has been deprecated in favor of "flat routes.  Please enable the new routing convention via the \`future.v2_routeConvention\` flag in your \`remix.config.js\` file.  For more information, please see https://remix.run/docs/en/main/file-conventions/route-files-v2.`;
```

Eventually I'd love to see it in yellow via `chalk` or something but I know we have future CLI improvements in the pipeline so probably worth waiting for that before doing anything too wild.

Reply:
I wonder if we should also make sure we have a section in the new docs on "here's how to continue using the old nested folders version via `defineRoutes`

Reply:
all uses of `warnOnce` could benefit from being yellow imo

- sessions not being signed
- cookies have expires setup
- flat routes v2
- serverBuildTarget